Final renaming of activities to events

stevejgordon opened the pull request:
Please merge after PR #722 (may require me to rebase this one before accepting it)

Includes:
File and folder renaming
Final source file updates
SQL file updates
CSS class updates

This final PR fixes issue #705

tonysurma commented:
this one can be rebased now

tonysurma commented:
is it me or are there several changes that aren't really changes... Did spacing change?

tonysurma commented on AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Events/EditEvent.cs:
```diff
@@ -2,240 +2,240 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using AllReady.Areas.Admin.Features.Events;
```
seems like most/all of the changes in this file aren't real changes?

stevejgordon commented:
@tonysurma I've rebased off of the latest master. I'm not sure why those files are showing complete changes. I renamed some folders which perhaps caused git to see complete file changes? I'm not really sure how to fix it though. Any thoughts?

tonysurma commented:
I unfortunately don't.  @BillWagner or @MisterJames any thoughts?  Maybe it is the folder name switch?

BillWagner commented:
I don't know. I hope to learn something from @MisterJames :smile:

stevejgordon commented:
I'm wondering if perhaps the line endings were different in some files and when I did my renaming of folders and final manual check of the files they got updated and saved.
